static-site/content/docs/07-visualisation/02-map-interpretation.md

```diff
@@ -0,0 +1,98 @@
+In the [zika dataset](https://nextstrain.org/zika) you can change this to "Region" and you'll see the map update accordingly.
+Each dataset is responsible for associating different locations with a GPS co-ordinate which we use as the center of the circle -- bear this in mind when you are interpreting the map and don't over-assign precision.
+For example, the circle representing the USA (labelled "1" in Figure 1) represents the entire country, not a specific location in Kansas!
+
+
+## Transmissions (Lines)
+
+These lines typically represent our best estimate of the movement of the pathogen over the selected time slice.
+For instance, the line indicated by "2" in Figure 1 indicates inferred movement of Zika virus from French Polynesia to Brazil.
+Each individual line represents a parent-child branch in the phylogenetic tree where the value of the current geographic-resolution differs.
+This means that if values for the geographic-resolution haven't been inferred for internal nodes of the tree then no transmission lines can be drawn.
+There is a toggle to switch these off in the sidebar (Figure 1, label "5"). 
+
+![](./img/labelled-movement.png)
+_Figure 3.
+Left: Cartoon of how the tree determines the construction of transmission lines.
+Right: Example of transmission lines showing [between country movement of Ebola](https://nextstrain.org/ebola)._
+
+**Line thickness.**
+Each rendered arc is actually a collection of individual lines of identical thickness (as above, each line identified by a branch in the tree indicating a change in the value of the current geographic resolution).
+Lines which look "thicker" therefore represent more observed cases of movement between the two locations (circles).
+For instance, in figure 3 the arc labelled by "1" is thicker than that labelled "3", indicating that in the [Ebola dataset](https://nextstrain.org/ebola) there are more observed cases of movement between Guinea and Sierra Leone than there are between Liberia and Guinea.
+
+
+**Directionality** is conveyed by the [orientation of the arc](https://en.wikipedia.org/wiki/Curve_orientation).
+A transmission line is a positively oriented curve when traveling from source to origin (parent node to child node on the tree). 
+In figure 3, line 1 is positively curved when moving from Guinea to Sierra Leone and therefore indicates the inferred viral movements from Guinea to Sierra Leone; line 2 represents movements from Sierra Leone to Guinea.
+
+
+**Line color** 
+It represents the currently selected "color-by" -- the metadata displayed in the legend -- of each transmission.
+If we examine the cartoon example in figure 3, we see that the parent and child nodes have a different location, which we will use to draw a line on the map from location A to location B. If the current color-by is "date", then the line will be colored blue, which corresponds to the metadata at the origin of the transmission.
```

Comment:
```suggestion
If we examine the cartoon example in Figure 3, we see that the parent and child nodes have a different location, which we will use to draw a line on the map from location A to location B. The color-by in this example is "date", so the line will be colored blue, which corresponds to the date value at the origin of the transmission.
```
The color helps me understand directionality better than the orientation, for what it's worth.

Edit: reading on, I see why color can't be the cue used for directionality:
> As transmission arcs potentially consist of a number of lines, we can have multiple different colors within a transmission arc between locations.